pysand/erosion.py

```diff
@@ -168,6 +224,16 @@ def reducer(v_m, rho_m, Q_s, D1, D2, d_p, GF=2, alpha=60, K=2e-9, n=2.6, rho_t=7
     :param rho_t: Target material density [kg/m3], default = 7850 (duplex steel)
     :return: Reducer erosion rate [mm/year]
     '''
+
+    # Unique input validation
+    for d in [D1, D2]:
```

Comment:
Do you think we should change parameter name from D1 to D? This is for consistency; that way you can include it also in the common validation... Otherwise we could say 'D':D1 in the kwargs dict. Also this could be part of the general validation function.